static/offline.html

```diff
@@ -13,7 +13,7 @@
         </style>
     </head>
     <body>
-    <div id="logo"><img src="static/images/logo_OL-err.png" alt="The Open Library is closed!" width="219"/></div>
+    <div id="logo"><img src="static/images/logo_OL-err.png" alt="The Open Library is closed!" width="220" height="140"/></div>
```

Comment:
Set it to the exact width/height + make the path absolute so it works when testing it at https://testing.openlibrary.org/static/offline.html

```suggestion
    <div id="logo"><img src="/static/images/logo_OL-err.png" alt="The Open Library is closed!" width="219" height="131"/></div>
```